code/modules/antagonists/cult/cult.dm

```diff
@@ -367,18 +368,24 @@
 	explanation_text = "Summon Nar'Sie by invoking the rune 'Summon Nar'Sie'. <b>The summoning can only be accomplished in [english_list(summon_spots)] - where the veil is weak enough for the ritual to begin.</b>"
 
 /datum/objective/eldergod/check_completion()
+	if(killed)
+		return -1 // You failed so hard that even the code went backwards.
 	return summoned || completed
 
 /datum/team/cult/proc/check_cult_victory()
 	for(var/datum/objective/O in objectives)
+		if(O.check_completion() == -1)
+			return -1
 		if(!O.check_completion())
-			return FALSE
-	return TRUE
+			return 0
+	return 1
 
 /datum/team/cult/roundend_report()
 	var/list/parts = list()
 
-	if(check_cult_victory())
+	if(check_cult_victory() == -1) // Epic failure, you summoned your god and then someone killed it.
```

Comment:
You could get away with calling `check_cult_victory()` only once here rather than twice

Reply:
Done, it's just not showing up.